### PR TITLE
solution rewrite

### PR DESCRIPTION
**Pull request by scottcreynolds**
change example, update tests, project structure, etc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "url": "https://github.com/learn-co-curriculum/js-ajax-callbacks-lab/issues"
+  },
+  "homepage": "https://github.com/learn-co-curriculum/js-ajax-callbacks-lab#readme",
+  "dependencies": {
+    "expect": "^1.20.2",
+    "handlebars": "^4.0.5",
+    "jquery": "^3.1.0",
+    "jsdom": "^9.4.1",
+    "mocha": "^2.5.3",
+    "mocha-jsdom": "^1.1.0",
```
Comment (pletcher) [inline package.json:35]: We'll need to make sure to `npm install -S mocha-multi` as well.

**Comment by pletcher** [comment]
This is great! Just wanna make sure that `mocha-multi` is in the package.json so that the Learn test runner can do its thing.

**Comment by scottcreynolds** [comment]
@pletcher all set

**Comment by pletcher** [comment]
Sweeeet

**Pull request merged**